LICENSE

```diff
@@ -1,4 +1,5 @@
-Copyright (c) 2016-2017 Pelagicore
+Copyright (c) 2016-2017 Pelagicore AB
```

Comment:
Just FYI we decided not to use the range way of writing, so if you see this in any files you can remove it and leave only the first year.

Reply:
Good to know. Do we update copyright with the current year or leave it as is?